### DISCUSSION TITLE
今年BW可能没办法用这个抢了

### DISCUSSION THREAD

**Discussion by KenshinZhan** [mikumifa/biliTickerBuy#115] [聊天专区]
这几天发现会员购网页版没法用这个抢票的，因为点进去买票页面是要你扫二维码去app抢票，这个只能网页抢票，七月萤火虫这些现在还能用这个抢，阿B你怎么这么坏啊😭

**Comment by mikumifa** [comment]
看一下#76

**Reply by KenshinZhan** [reply]
感谢

**Reply by KenshinZhan** [reply]
不过感觉还是得做好两手准备吧，不然以阿B的尿性不知道又要搞什么幺蛾子😢

**Reply by mikumifa** [reply]
bw应该不止1轮，第一轮如果没啥问题的话，就放心抢了，第一轮还是小心点
